src/app/application.cpp

```diff
@@ -815,6 +819,65 @@ void Application::applyMemoryWorkingSetLimit() const
 #endif
 
 #ifdef Q_OS_WIN
+MemoryPriority Application::getProcessMemoryPriority() const
+{
+    return m_processMemoryPriority.get(MemoryPriority::Normal);
```

Comment:
To be honest, following the previous discussions, I thought that you would go the other way, i.e. forcibly set the normal priority to the main thread when the process is set to a lower priority.

Reply:
Oh, I was testing a few combinations and forgot to try this one... This seems better than reverting all threads to normal.
PR updated.